variant_remapping_tools/tests/test_reverse_strand.py

```diff
@@ -0,0 +1,80 @@
+from collections import Counter
+from unittest import TestCase
+from unittest.mock import Mock
+
+import pysam
+
+from variant_remapping_tools.reverse_strand import calculate_variant_position, is_read_valid
+
+
+class TestReverseStrand(TestCase):
+
+    def _mock_read(self, pos=1000, is_unmapped=False, is_secondary=False, cigartuples=((pysam.CMATCH, 101),), tags=None):
+        if tags is None:
+            tags = {'AS': -6}
+        read = Mock(pos=pos, is_unmapped=is_unmapped, is_secondary=is_secondary, cigartuples=cigartuples)
+        read.get_tag = lambda x: tags[x]
+        return read
+
+    def setUp(self) -> None:
+        self.counter = Counter()
+        self.score_cutoff = -20
+        self.diff_cutoff = .06
+
+    def test_calculate_variant_position(self):
+        # 101M: Perfect alignment
```

Comment:
I think if you convert comments to one-line docstrings, pytest will be able to automatically use them for test "descriptions" and error messages in case something fails

Reply:
I did not know that. That a good trick.